src/modules/initcpiocfg/main.py

```diff
@@ -180,10 +189,11 @@ def modify_mkinitcpio_conf(partitions, root_mount_point):
     else:
         hooks.extend(["filesystems"])
 
-    if btrfs == "yes" and cpu['proc0']['vendor_id'].lower() != "genuineintel":
+    cpu_intel = "genuineintel"
```

Comment:
The following code becomes shorter if you do `cpu_is_intel = (cpu["proc0"]["vendor_id"].lower() == "genuineintel")` here.